### ISSUE TITLE
Forced to use a very low batch_size

### ISSUE THREAD

**Issue by vignesh10** [facebook/C3D#39]
Hi,

While training/ finetuning my model, I am unable to increase the batch_size more than a value of 5. However I am able to have a batch_size of value 30 while testing. 
I am running my scripts on Titan Black GPU. 

What could be the reason for such a behavior?

**Comment by dutran** [comment]
Training always requires more memory compared to testing. Extra memory are needed for gradients in backprop.
